x-pack/test/functional/apps/monitoring/elasticsearch/nodes_mb.js

```diff
@@ -189,19 +189,21 @@ export default function ({ getService, getPageObjects }) {
         });
       });
 
-      // this is actually broken, see https://github.com/elastic/kibana/issues/122338
-      it.skip('should sort by status', async () => {
-        const sortedStatusesAscending = ['Status: Offline', 'Status: Online', 'Status: Online'];
-        const sortedStatusesDescending = [...sortedStatusesAscending].reverse();
-
+      it('should sort by status', async () => {
         await nodesList.clickStatusCol();
-        await retry.try(async () => {
-          expect(await nodesList.getNodeStatuses()).to.eql(sortedStatusesDescending);
-        });
-
         await nodesList.clickStatusCol();
+
+        // retry in case the table hasn't had time to re-render
         await retry.try(async () => {
-          expect(await nodesList.getNodeStatuses()).to.eql(sortedStatusesAscending);
+          const nodesAll = await nodesList.getNodesAll();
+          const tableData = [
+            { status: 'Status: Online' },
+            { status: 'Status: Online' },
+            { status: 'Status: Offline' },
+          ];
+          nodesAll.forEach((obj, node) => {
+            expect(nodesAll[node].status).to.be(tableData[node].status);
+          });
```

Comment:
Nitpick: Minor naming thing.
```suggestion
          nodes.forEach((_, index) => {
            expect(nodes[index].status).to.be(expectedNodes[index].status);
          });
```